src/core/datatypes/phylogenetics/character/DiscretizedContinuousState.cpp

```diff
@@ -0,0 +1,320 @@
+#include "DiscretizedContinuousState.h"
+
+#include <boost/lexical_cast.hpp>
+#include <string>
+
+#include "RbException.h"
+#include "Cloneable.h"
+
+using namespace RevBayesCore;
+
+/** Default constructor */
+DiscretizedContinuousState::DiscretizedContinuousState(size_t n, double d) : DiscreteCharacterState( n ),
+    is_gap( false ),
```

Comment:
For a lot of these default values, it might be best to just define `bool is_gap = false` in the class declaration.

Reply:
Up to you though.

Reply:
I had no idea you could do this! I've updated this class with defaults in the header file. Thanks for pointing out that we can do that!